src/legacy/components/batch-models-multi-select/CopyInUserBatchModelsMultiSelect.component.js

```diff
@@ -211,14 +188,15 @@ export default class CopyInUserBatchModelsMultiSelectComponent extends React.Com
             .value();
 
         return (
-            <Dialog
+            <ConfirmationDialog
                 title={title}
-                actions={this.getDialogButtons()}
-                autoScrollBodyContent
-                autoDetectWindowHeight
-                contentStyle={this.styles.dialog}
+                maxWidth={"lg"}
+                fullWidth={true}
                 open={true}
-                onRequestClose={this.props.onRequestClose}
+                onCancel={this.props.onCancel}
+                cancelText={this.getTranslation("cancel")}
+                infoActionText={this.getTranslation("save")}
+                onInfoAction={!isLoading ? this.copy.bind(this) : undefined}
```

Comment:
```suggestion
                saveText={this.getTranslation("save")}
                onSave={!isLoading ? this.copy.bind(this) : undefined}
```

@jvfdunkley For future PRs